.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
+root = true
+
+[*]
```

Comment:
This could introduce unwanted side effects.
I would start a bit more conservative and restrict it to something like [*.{php,css,js,sh,md}]
At least UTF-8 should be restricted, maybe even more.
e.g. what will happen if you store a file like api/soap/mantisconnect.wsdl that comes with ISO-8859-1 encoding?